core/client/fs/src/main/java/alluxio/client/file/options/OutStreamOptions.java

```diff
@@ -437,15 +406,14 @@ public int hashCode() {
   @Override
   public String toString() {
     return MoreObjects.toStringHelper(this)
+        .add("common-options", mCommonOptions)
```

Comment:
could we do `commonOptions` to follow the pattern for the rest of the toString items? And also put this in alphabetical order?

what does the `toString` of `mCommonOptions` look like?

Reply:
It's pretty much harmonious with MoreObjects.toStringHelper.
You can check any RPC log to see what it looks like.